About/whitepaper.md

```diff
@@ -65,14 +65,13 @@ untrusted components in a system, including full virtual machines.
 {% svg /images/tries-shutterstock/sel4-principles.drawio.svg width="100%" %}
 </div>
 
-In the diagram, the seL4 kernel is the layer directly above the hardware,
-mediating access to it. On top of seL4, this example is running a virtual
-machine with untrusted code, some native applications with untrusted code, and
-some native applications with critical trusted code. The kernel isolates trusted
-from untrusted code and provides secure, controlled communication mechanisms
-between them. In the example, the system is configured to only allow
-communication via the trusted application, which for instance could implement
-custom security policies.
+In the diagram, the seL4 kernel is the layer directly above the hardware and
+mediates access to it. This example shows a virtual machine running on top of
+seL4. Some native applications in the virtual machine have untrusted code, and
```

Comment:
The flow between the first and second sentence doesn't work now. The subject of the first sentence is seL4. The second sentence needs to follow on from something in the first sentence, not start something fresh ("this example"). The (now) third sentence isn't accurate -- the applications are not inside the VM, but outside the VM, next to it (which means that the original text didn't bring that across and does need to change).

Reply:
I also prefer the original version for this one.

Reply:
Righteo. So when you say "On top of seL4, this example is running a virtual machine with untrusted code, some native applications with untrusted code, and some have critical trusted code."
do you mean that the "example is running is running a .... on top of seL4"?
Sorry if I'm being a pain in the arse. I'm trying to learn as I go, as well as proofread. :-)

Reply:
> do you mean that the "example is running is running a .... on top of seL4"?

Yes, that's right. And I think I'm getting now what the problem with the original was. I might try to push an update for that one.

> Sorry if I'm being a pain in the arse. I'm trying to learn as I go, as well as proofread. :-)

No it's good -- it's very hard to figure out what doesn't come across clearly, and this is good feedback. We may not always like it, but it's needed :-)